vagrant/Install-on-Centos-8.sh

```diff
@@ -26,17 +26,17 @@
 # Now you can install all packages needed for Nominatim:
 
 #DOCS:    :::sh
-    sudo dnf --enablerepo=PowerTools install -y postgresql10-server \
-                        postgresql10-contrib postgresql10-devel postgis25_10 \
+    sudo dnf --enablerepo=PowerTools install -y postgresql12-server \
+                        postgresql12-contrib postgresql12-devel postgis30_12 \
                         wget git cmake make gcc gcc-c++ libtool policycoreutils-python-utils \
                         llvm-toolset ccache clang-tools-extra \
                         php-pgsql php php-intl php-json libpq-devel \
-                        proj52-epsg bzip2-devel proj-devel boost-devel \
+                        bzip2-devel proj-devel boost-devel \
```

Comment:
I'd expect that proj-devel can be removed as well.

Reply:
Looks like proj-devel is required by Osmium

```    centos8: CMake Warning at osm2pgsql/cmake/FindOsmium.cmake:212 (message):
    centos8:   Osmium: PROJ.4 library is required but not found, please install it or
    centos8:   configure the paths.
    centos8: Call Stack (most recent call first):
    centos8:   osm2pgsql/CMakeLists.txt:91 (find_package)
    centos8:
    centos8: CMake Error at /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
    centos8:   Could NOT find Osmium (missing: PROJ_INCLUDE_DIR PROJ_LIBRARY) (found
    centos8:   suitable version "2.15.4", minimum required is "2.14")
    centos8: Call Stack (most recent call first):
    centos8:   /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:378 (_FPHSA_FAILURE_MESSAGE)
    centos8:   osm2pgsql/cmake/FindOsmium.cmake:281 (find_package_handle_standard_args)
    centos8:   osm2pgsql/CMakeLists.txt:91 (find_package)
    centos8:```

Reply:
Oh dear, looks like we haven't updated osm2pgsql in a while.